pyramda/iterable/reject.py

```diff
@@ -0,0 +1,14 @@
+from pyramda.function.curry import curry
+from . import filter
+
+
+@curry
+def reject(p, xs):
+    """
+    Acts as a complement  of `filter`
+
+    :param p: predicate
+    :param xs: Iterable. A sequence, a container which supports iteration or an iterator
+    :return: list
+    """
+    return list(set(xs) - set(filter(p, xs)))
```

Comment:
That for comprehension will (I think) filter the list once for each loop iteration, making the operation take quadratic time.

I think we should just reuse the `filter` and `complement` functions defined elsewhere in Pyramda, like so:

```python
def reject(p, xs):
  return filter(complement(p), xs)
```